pkg/render/logstorage.go

```diff
@@ -660,10 +661,10 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 			es.installation.CertificateManagement,
 			es.csrImage,
 			csrVolumeNameHTTP,
-			ElasticsearchServiceName,
+			ElasticsearchInternalServiceName,
```

Comment:
This change will have some consequences for our users that upgrade elasticsearch. What will happen is that alongside the current Elasticsearch CR, a new CR will be created with this new name. That means that the user will have two clusters alongside, one with the old data and one that is empty. I believe this cluster will stay empty, no current data will be ported over. Also, the user needs to have enough disks for both clusters. We need to come up with a solution if we want to keep this change, which might involve adding extra upgrade steps to the docs.